src/Component/CountClass.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+let rand= Math.floor(Math.random()*100)
+
+export default class child extends Component {
+    constructor(){
+        super()
+        this.state={
+            count:rand,
+        }
+    }
+    counter(value){
+        value===0 ? this.setState({count:this.state.count -1}): this.setState({ count: this.state.count + 1 })
```

Comment:
== will be enough.
optimize this line of  code  by using single setState({})
let count = value==0 ? this.state.count -1 : this.state.count + 1;
this.setState({count})